test/javascript/components/offcanvas.js

```diff
@@ -104,7 +104,7 @@ describe('Off Canvas', function() {
           plugin.$element.should.have.class('is-open');
           $('body').should.have.class('is-off-canvas-open');
           done();
-        }, 1);
+        }, 80);
```

Comment:
Why this ?

Reply:
Because it is too short for CI and produced a failing test.

Reply:
I general there was a race condition which gave `null`.

Reply:
Mh, testing. Might be another issue there =/

Reply:
We might have to change this there as both run in parallel as it seems (the event and the other code part).

Reply:
> In general there was a race condition which gave null.

I don't understand what you're talking about

> Because it is too short for CI and produced a failing test.

I think this is caused by the `opened.zf.offcanvas` event being triggered _before_ the class and everything is set, which is not a good thing (because the offcanvas is not fully `opened`).

We should resolve that instead of putting a workarounds in the tests.